packages/store-ui/src/molecules/Carousel/Carousel.tsx

```diff
@@ -86,12 +86,7 @@ function Carousel({
   const slides = preRenderedSlides.concat(children ?? [], postRenderedSlides)
 
   return (
-    <section
-      id="store-carousel"
-      data-store-carousel
-      data-testid={testId}
-      aria-label="carousel"
-    >
+    <section data-store-carousel data-testid={testId} aria-label="carousel">
```

Comment:
This is actually necessary since we're using `aria-controls` on the arrows below. Without the `id` you should get another a11y error